.env.example

```diff
@@ -24,13 +24,13 @@ AUTH_SOLANA_LOGIN_ENABLED=false
 # Enable register with Solana
 AUTH_SOLANA_REGISTER_ENABLED=false
 # Telegram Bot Name
-AUTH_TELEGRAM_BOT_NAME=
+#AUTH_TELEGRAM_BOT_NAME=
```

Comment:
We disable these options by default. Especially with Telegram as it's in an early stage.